MIT.md

```diff
@@ -1,10 +1,7 @@
-## Copyright 2021, [YOUR NAME]
```

Comment:
-Pleas remove this part because you have already mentioned this part.

Reply:
- [ ] Please when you give a suggestion make sure you select the right part, for example here you ask to remove a part that is already removed 🙃 🙃
- [ ] Please make sure to read carefully your comment before submitting it to avoid typo errors 🙃 🙃, also use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.